.github/README.md

```diff
@@ -0,0 +1,37 @@
+# GSRI Zeus Manager
+
+## What is GSRI Zeus Manager
+It is a set of functions designed to rely on CBA custom command registering in order to interact with any Zeus module added in 3Den without having to link them independently.
```

Comment:
GZM is a set of functions allowing to handle who may become Zeus on your mission, based on a server whitelist instead of through mission modules. You can change who can Zeus you mission, directly from the chat, without updating the mission itself, nor without having to grant admin privileges to them.